### ISSUE TITLE
Infinite loading spinner on component preview

### ISSUE THREAD

**Issue by kongweiying2** [fwouts/previewjs#1983]
### Describe the bug

https://vscode.dev/github/Eusaybia/lifemap/blob/calculator/src/view/content/Math.tsx

I'm getting an infinite spinner when I'm trying to load my components, even on the simplest projects, with a single React Fragment.

I'm on React 18.2.0 and have upgraded to the latest version of VSCode so that doesn't seem to be the problem. 

### Reproduction

1. Go to this repo https://vscode.dev/github/Eusaybia/lifemap/blob/calculator/src/view/content/Math.tsx
2. Go to the Message.tsx file
3. Click on Open Message in Preview.js

It should show an infinite spinner

### Preview.js version

v1.24.2

### Framework

React 18.0.2

### System Info

```shell
npx envinfo --system --binaries --ides --browsers
```

### Used Package Manager

yarn

### Extension logs (useful for crashes)

```shell
2023-09-04 14:43:15.161 [info] Extension host with pid 66353 started
2023-09-04 14:43:15.234 [info] ExtensionService#_doActivateExtension vscode.git-base, startup: true, activationEvent: '*', root cause: vscode.git
2023-09-04 14:43:15.238 [info] ExtensionService#_doActivateExtension vscodevim.vim, startup: true, activationEvent: '*'
2023-09-04 14:43:15.272 [info] ExtensionService#_doActivateExtension vscode.github-authentication, startup: false, activationEvent: 'onAuthenticationRequest:github'
2023-09-04 14:43:15.297 [info] ExtensionService#_doActivateExtension vscode.git, startup: true, activationEvent: '*'
2023-09-04 14:43:15.309 [info] ExtensionService#_doActivateExtension vscode.github, startup: true, activationEvent: '*'
2023-09-04 14:43:15.337 [info] ExtensionService#_doActivateExtension vscode.microsoft-authentication, startup: false, activationEvent: 'onAuthenticationRequest:microsoft'
2023-09-04 14:43:15.408 [info] Eager extensions activated
2023-09-04 14:43:15.416 [info] ExtensionService#_doActivateExtension vscode.debug-auto-launch, startup: false, activationEvent: 'onStartupFinished'
2023-09-04 14:43:15.418 [info] ExtensionService#_doActivateExtension vscode.merge-conflict, startup: false, activationEvent: 'onStartupFinished'
2023-09-04 14:43:15.422 [info] ExtensionService#_doActivateExtension GitHub.vscode-pull-request-github, startup: false, activationEvent: 'onStartupFinished'
2023-09-04 14:43:15.442 [info] ExtensionService#_doActivateExtension zenclabs.previewjs, startup: false, activationEvent: 'onStartupFinished'
2023-09-04 14:43:19.177 [info] Extension host terminating: renderer closed the MessagePort
2023-09-04 14:43:19.183 [error] Canceled: Canceled
	at new h (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:102:56634)
	at t.O (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:114:8020)
	at N.<computed>.T.charCodeAt.N.<computed> (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:114:5459)
	at a.g (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:102:77241)
	at a.executeCommand (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:102:76679)
	at v.registerCommand.description (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:102:76000)
	at a.h (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:102:77777)
	at a.g (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:102:76772)
	at a.executeCommand (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:102:76679)
	at Object.executeCommand (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:120:20977)
	at r0.reset (/Users/kongweiying/.vscode/extensions/github.vscode-pull-request-github-0.70.0/dist/extension.js:1822:4008)
	at r0.dispose (/Users/kongweiying/.vscode/extensions/github.vscode-pull-request-github-0.70.0/dist/extension.js:1822:4247)
	at S (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:13:11731)
	at r.Z (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:116:9686)
	at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:116:7582
	at Array.map (<anonymous>)
	at r.S (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:116:7570)
	at r.terminate (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:116:7842)
	at s.terminate (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:120:1506)
	at C (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:130:32254)
	at MessagePortMain.<anonymous> (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:130:28679)
	at MessagePortMain.emit (node:events:513:28)
	at MessagePortMain._internalPort.emit (node:electron/js2c/utility_init:2:367) setContext undefined
2023-09-04 14:43:19.188 [info] Extension host with pid 66353 exiting with code 0
2023-09-04 14:43:19.732 [info] Extension host with pid 66401 started
2023-09-04 14:43:19.732 [info] Skipping acquiring lock for /Users/kongweiying/Library/Application Support/Code/User/workspaceStorage/7b95e387464df1a08e1750736e05b71c.
2023-09-04 14:43:19.775 [info] ExtensionService#_doActivateExtension vscode.emmet, startup: false, activationEvent: 'onLanguage'
2023-09-04 14:43:19.781 [info] ExtensionService#_doActivateExtension vscode.typescript-language-features, startup: false, activationEvent: 'onLanguage:typescriptreact'
2023-09-04 14:43:19.791 [info] ExtensionService#_doActivateExtension zenclabs.previewjs, startup: false, activationEvent: 'onLanguage:typescriptreact'
2023-09-04 14:43:19.903 [info] ExtensionService#_doActivateExtension vscode.git-base, startup: true, activationEvent: '*', root cause: vscode.git
2023-09-04 14:43:19.904 [info] ExtensionService#_doActivateExtension vscodevim.vim, startup: true, activationEvent: '*'
2023-09-04 14:43:19.938 [info] ExtensionService#_doActivateExtension vscode.npm, startup: true, activationEvent: 'workspaceContains:package.json'
2023-09-04 14:43:20.136 [info] ExtensionService#_doActivateExtension vscode.git, startup: true, activationEvent: '*'
2023-09-04 14:43:20.154 [info] ExtensionService#_doActivateExtension vscode.github, startup: true, activationEvent: '*'
2023-09-04 14:43:20.344 [info] Eager extensions activated
2023-09-04 14:43:20.409 [info] ExtensionService#_doActivateExtension vscode.debug-auto-launch, startup: false, activationEvent: 'onStartupFinished'
2023-09-04 14:43:20.411 [info] ExtensionService#_doActivateExtension vscode.merge-conflict, startup: false, activationEvent: 'onStartupFinished'
2023-09-04 14:43:20.414 [info] ExtensionService#_doActivateExtension vscode.github-authentication, startup: false, activationEvent: 'onStartupFinished', root cause: GitHub.vscode-pull-request-github
2023-09-04 14:43:20.461 [info] ExtensionService#_doActivateExtension GitHub.vscode-pull-request-github, startup: false, activationEvent: 'onStartupFinished'
2023-09-04 14:43:20.782 [info] ExtensionService#_doActivateExtension vscode.microsoft-authentication, startup: false, activationEvent: 'onAuthenticationRequest:microsoft'
```

**Comment by kongweiying2** [comment]
Actually, I just downgraded the extension and then upgraded and it seems to work again now. Strange!

**Comment by fwouts** [comment]
Not sure what happened here. I've just released v1.24.3, let me know if you see this happen again.
